src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -897,6 +897,7 @@ private function addValidationSection(ArrayNodeDefinition $rootNode, callable $e
                             ->validate()->castToArray()->end()
                         ->end()
                         ->scalarNode('translation_domain')->defaultValue('validators')->end()
+                        ->scalarNode('password_strength_estimator')->defaultValue('validator.default_password_strength_estimator')->end()
```

Comment:
If we use the class name as service name, it would be easier for a developer to find the default implementation class.

Reply:
OK I will change it

Reply:
That would be unusual for services defined by FrameworkBundle. I'd keep it as it is.

Reply:
What about having an alias?

Reply:
Using the class name as id makes this class name autowirable. That's not what we want (especially not for an implementation of an interface)

Reply:
So we can add the interface name as an alias for the service. So that developers can redefine the alias to their own implementation, instead of using the configuration.

Reply:
I'd prefer keeping things simple. My recommendation would be to:
- remove this config option
- create an alias named `validator.password_strength_estimator` that maps to `validator.default_password_strength_estimator`. That's enough if ppl want to replace the implementation. No need to mess up with autowiring aliases either.

Reply:
I removed the config options and creted the service/alias.